doc/releases/quincy.rst

```diff
@@ -68,7 +68,7 @@ General
 
 * MGR: The progress module disables the pg recovery event by default since the
   event is expensive and has interrupted other services when there are OSDs
-  being marked in/out from the the cluster. However, the user can still enable
```

Comment:
This kind of thing (looking for instances of double "the") is the kind of thing I gratefully welcome into the auto-correct workflow.